introduction.md

```diff
@@ -0,0 +1 @@
+Hi, I'm working on the "Mastering Github" course on CodeSchool. This project, dojo_rules, is for completing lesson excercises.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.